openrtb_ext/request.go

```diff
@@ -0,0 +1,28 @@
+package openrtb_ext
+
+// ExtRequest defines the contract for bidrequest.ext
+type ExtRequest struct {
+	Prebid ExtRequestPrebid `json:"prebid"`
+}
+
+// ExtRequestPrebid defines the contract for bidrequest.ext.prebid
+type ExtRequestPrebid struct {
+	Targeting *ExtRequestTargeting`json:"targeting"`
+}
+
+// ExtRequestTargeting defines the contract for bidrequest.ext.prebid.targeting
+type ExtRequestTargeting struct {
+	PriceGranularity PriceGranularity `json:"pricegranularity"`
+	MaxLength        int              `json:"lengthmax"`
+}
+
+// PriceGranularity defines the allowed values for bidrequest.ext.targeting.pricegranularity
```

Comment:
Shouldn't it be `bidrequest.ext.prebid.targeting.pricegranularity`?

Reply:
Yes! good catch... I'll fix it.